Cargo.toml

```diff
@@ -43,7 +43,7 @@ rand = { version = "0.8" }
 log = { version = "0.4", optional = true }
 # WASM API
 wasm-bindgen = { version = "0.2.87", optional = true }
-hax-lib = { git = "https://github.com/hacspec/hax/", branch = "karthik/core-for-kyber" }
+hax-lib = { git = "https://github.com/hacspec/hax/", branch = "main" }
```

Comment:
We shouldn't need a branch here?

Reply:
I get the following error when I remove the branch:

```
error: failed to download `hax-lib v0.1.0-pre.1 (https://github.com/hacspec/hax/#f31b319e)`

Caused by:
  unable to get packages from source

Caused by:
  failed to find hax-lib v0.1.0-pre.1 (https://github.com/hacspec/hax/#f31b319e) in path source
note: this is an unexpected cargo internal error
note: we would appreciate a bug report: https://github.com/rust-lang/cargo/issues/
note: cargo 1.76.0-nightly (6790a5127 2023-11-10)
```